cirq/circuits/optimization_pass.py

```diff
@@ -131,7 +112,7 @@ def optimization_at(self,
         """
         pass
 
-    def optimize_circuit(self, circuit: Circuit):
+    def __call__(self, circuit: Circuit):
```

Comment:
I would keep the `optimize_circuit` method and have `__call__` delegate to it. It's a form of documentation and API-discovery-by-autocomplete.